Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14
```

Comment:
I believe this was mentioned previously, but I don't remember- is 14 explicitly required? I generally prefer to use `node:lts`, which moves from Node 12 to 14 next month, and ideally `node:lts-alpine` to reduce attack surface area. We may not explicitly need the reduced attack surface area for this example project, but I do want us to push best practices in everything we publish.

Reply:
Are you happy enough with `node:14-alpine`? I take your point but would prefer to specify the exact version of Node in this case.

Reply:
That works great 👍